src/App.css

```
@@ -746,3 +746,185 @@ table.not-prose th {
+    padding: 0.75in 1in !important;
+    margin: 0 !important;
+  }
+
+  /* Ensure page breaks work well */
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    page-break-after: avoid;
```

Stylelint (17.2.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)